src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -1760,7 +1769,12 @@ public void setNewTypeInference(boolean enable) {
     this.useNewTypeInference = enable;
   }
 
-  /**
+  // Not dead code; used by the open-source users of the compiler.
```

Comment:
You sure? It's a note rather than docs about the method. See https://github.com/visokio/closure-compiler/blame/allowOtiAndNtiWarnings/src/com/google/javascript/jscomp/CompilerOptions.java#L101